homeassistant/components/evohome/__init__.py

```diff
@@ -24,7 +40,7 @@
 from homeassistant.core import HomeAssistant, ServiceCall, callback
 from homeassistant.helpers import entity_registry as er
 from homeassistant.helpers.aiohttp_client import async_get_clientsession
-import homeassistant.helpers.config_validation as cv
+import homeassistant.helpers.config_validation as cv  # noqa: E402
```

Comment:
I'm curious: why is an E402 thrown here?

Reply:
It shouldn't have - it doesn't any more.